src/pages/myGeotab/guides/codeBase/usingInDotnet.tsx

```diff
@@ -0,0 +1,164 @@
+    <p>The last parameter to this Call method is an <a href="http://msdn.microsoft.com/en-us/library/bb397696.aspx" target="_blank" rel="noopener noreferrer">anonymous object</a> which contains the 
+    parameters for the method (please review the <a href="../../api/reference/#Remove1">API reference</a> to see which parameters the method expects, and whether the parameters are required or optional). 
+    The parameter order is not significant, and it is acceptable to omit optional parameters. Optional parameters will revert to their default values, typically "null" or “false” values.</p>
+    <p>The API class automatically handles databases that are moved to different servers in the federation and expired tokens (token are typically valid for 2 weeks) by automatically
+        re-authenticating and continuing.</p>
+    <h2>Example Code</h2>
+    <p>The following is a simple console app that will output the latitude and longitude of each device in a list of devices:</p>
+    <CodeSample
+        language="csharp"
+        code={
+            `// Create the API object and authenticate
+public static async Task Main(string[] args) {
+// Create the API object and authenticate
+    var api = new API("bob@geotab.com", "password", null, "demo", "server");
+    await api.AuthenticateAsync();
+            
+    // Get all devices
+    var devices = await api.CallAsync<List<Device>>("Get", typeof(Device));
+    Console.WriteLine("SerialNumber\tLatitude\tLongitude");
+    foreach (Device device in devices) {
+        // Get the Device Status Info which contains the current latitude and longitude for this device
+            var results = await api.CallAsync<List<DeviceStatusInfo>>("Get", typeof(DeviceStatusInfo), new {
+                search = new DeviceStatusInfoSearch {
+                    DeviceSearch = new DeviceSearch {
+                        Id = device.Id
+                    }
+                }
+            });
+            
+        if (results.Count <= 0) {
+            continue;
+        }
+            
+        DeviceStatusInfo deviceStatus = results[0];
+            
+        // Print the results to the console
+        Console.WriteLine(device.SerialNumber + "\t" + deviceStatus.Latitude + "\t" + deviceStatus.Longitude);
+    }
+}`
+        }
+    ></CodeSample>
+    <InformationalBox>
+        <p>Hint: async main method requires <a href="https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-7-1" target="_blank" rel="noopener noreferrer">C# 7.1</a>.</p>
+    </InformationalBox>
+    <p>After entering your credentials into the API constructor, this example will produce results similar to this in your command prompt window.</p>
+    <CodeSample
+        language="bash"
+        code={
+            `> dotnet run
+SerialNumber    Latitude        Longitude
+G70000000001    43.4371071      -79.7124329
+G70000000002    43.3683701      -79.7784042
+G80000000003    43.4620934      -79.6879883`
+        }
+    ></CodeSample>
```

Comment:
This can be self-closing since there is no content inside the tags.
```suggestion
    />
```